internal/server/proto/server.proto

```diff
@@ -58,6 +58,9 @@ service Waypoint {
   // ListInstances returns the running instances of deployments.
   rpc ListInstances(ListInstancesRequest) returns (ListInstancesResponse);
 
+  // FindExecInstance returns an instance that can be used for an exec session.
```

Comment:
Actually not sure what the use case is for this, can you address that in the comment?